predicators/ground_truth_models/spot_env/options.py

```diff
@@ -494,7 +494,8 @@ def _place_object_on_top_policy(state: State, memory: Dict,
     # Special case: the robot is already on top of the surface (because it is
     # probably the floor). When this happens, just drop the object.
     surface_geom = object_to_top_down_geom(surface_obj, state)
-    if surface_geom.contains_point(robot_pose.x, robot_pose.y):
+    if surface_geom.contains_point(
+            robot_pose.x, robot_pose.y) and surface_obj.name == "floor":
```

Comment:
what happened here? just curious

Reply:
oh - sometimes it appears the robot gets on top of the table in simulation?